tests/trainer/test_fsdp.py

```diff
@@ -24,7 +24,7 @@ def test_fsdp_device_initialization(model: ComposerClassifier, device: str):
     'meta' devices. This is because 'meta' will result in deferred initialization until FSDP is initialized
 
     """
-    num_classes = 10
+    num_classes = 2
```

Comment:
y?

Reply:
lol sorry I was debugging and didn't change it back